openapi/paths/friendships@{friendshipId}@request-events@messages.yaml

```diff
@@ -0,0 +1,34 @@
+get:
+  operationId: getRequestEventsMessages
+  summary: Gets the list of sent messages of incoming and outgoing friendship request events.
+  tags:
+    - History
+    - Requests
```

Comment:
The endpoint retrieves messages, so it seems strange to use tags such as `History` and `Requests`. However, I am not fond of tags like 'Messaging' either. What do u think would be a better alternative?

Reply:
I don't really know what are we using tags for, but in case they are needed I'd use something similar to the URL, for example friendships, and requests.

Reply:
They are used to organize the endpoints into "menus" as shown in the img:

<img width="262" alt="image" src="https://user-images.githubusercontent.com/42394626/216683917-99e17e65-9a05-46d6-be11-47a1dd97875a.png">

In this case the new endpoint is located under both the `History` and `Requests` menus.

Reply:
mm I don't like exposing twice and endpoint, it should be deterministic 🤔

Reply:
Okay, I'll go with the `Requests` tag.

Reply:
Should I refactor the tags for the [incoming and outgoing friendship requests](https://social-service-api-specs.pages.dev/#tag/Requests/operation/getOwnFriendRequests) endpoint? As in that case the endpoint is located under both `Friends` and `Requests` menus.